pkgs/by-name/xw/xwax/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  SDL,
+  SDL_ttf,
+  jack2,
+  libmpg123,
+  ffmpeg,
+  cdparanoia,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xwax";
+  version = "1.9";
+
+  src = fetchurl {
+    url = "https://xwax.org/releases/xwax-${finalAttrs.version}.tar.gz";
+    hash = "sha256-m+2PoUMYKBhlA2H0kld1W/iR8UMWEGaqp7yoxszp9jI=";
+  };
+
```

```suggestion
  postPatch = ''
    substituteInPlace Makefile \
      --replace-fail sdl-config "${lib.getExe' (lib.getDev SDL) "sdl-config"}"
  '';
```
When cross-compiling sdl-config will not be available in $PATH, so we must provide the full path.